oeps/processes/oep-0057-core-repositories.rst

```diff
@@ -0,0 +1,166 @@
+
+* establish the purpose of the openedx GitHub organization,
+* define a transparent process for the management of the organization's set of repositories, and
+* explain how community members can prepare their repositories for inclusion in the organization.
+
+Definition of Core Repositories
+-------------------------------
+
+* We define the *Core repositories of Open edX* (or in this context, "the Core") as the entire set of public repositories hosted in the `openedx GitHub organization`_.
+
+  * Phrased another way: A repository is in the Core if and only if it is public and in the openedx GitHub organization.
+
+  * Terminology notes:
+
+    * The choice of *Core* here is inspired by the Core Contributor program, which exists in part to grant community members access to repositories in the openedx GitHub organization.
+
+    * This does not preclude other, smaller "cores" of Open edX (such as a *learning core*, *core utilities*, etc.) from existing within the Core repositories.
+
+  * **What repositories are in the Core?**
+
+    * A repository *must* be in the Core if it is directly included in `latest Open edX release`_.
+
+    * A repository *must* be in the Core if:
+
+      * it is listed as a non-optional base dependency of a repository included in the latest Open edX release *and*
+
+      * it is *not* maintained as a third-party package that is unspecific to Open edX, such as Django or Python Requests.
+
+    * In addition to the repositories that must be in the Core, a repository *may* be in the Core if:
+
+      * it is of interest to more than one entity the Open edX community (TODO: this needs better criteria); and
+
+      * it has assigned `maintainers`_ who strive to:
+
+        * review community code contributions,
+        * participate in community discussions related to the repository, and
+        * keep the repository compliant with the accepted Open edX Proposals (OEPs).
+
+* We declare that all access to write, configure, or otherwise administer Core repositories shall be granted through the `Core Contributor Program`_ and managed by its Program Administrators.
+
+* We declare that the addition or removal of repositories from the Core should by reviewed by existing Core Contributors.
+
+  * TODO: Define a review process.
+
+
+Reasoning
+---------
+
+Many other repositories exist as part of the Open edX community. Some of these are forks of Core repositories, and others are upstream repositories in their own right. These various non-Core repositories are vital to different Open edX community members. Furthermore, the existence of non-Core repositories in general is important to the health of the Open edX project, which thrives on collaboration, experimentation, and extension.
+
+However, from these many repositories, we wish to enumerate a smaller set which:
+
+* is a good target for enhancement, documentation, and technical investment;
```

Comment:
nit: before the bullets you use `repositories... which`, so your verbs at the beginning of bullets should be plural. eg, s/is a good target/are good targets/

Reply:
Ah, got it. Yeah for some reason it does read awkwardly (to me); I actually think then the problem is the `which`. "we wish to enumerate a smaller set **which** is a good target" vs "we wish to enumerate a smaller set **that** is a good target"

Reply:
I agree but also agree it's not a blocker. Offered a suggestion if you want to fold it in, @kdmccormick